pom.xml

```diff
@@ -6,7 +6,7 @@
 
     <groupId>io.citrine</groupId>
     <artifactId>lolo</artifactId>
-    <version>2.2.5</version>
+    <version>0.2.9</version>
```

Comment:
2.3.0?

Reply:
This is a PR into develop, so it shouldn't be changing at all.